ui/test/kapacitor/components/KapacitorRules.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import {shallow} from 'enzyme'
+
+import _ from 'lodash'
+
+import KapacitorRules from 'src/kapacitor/components/KapacitorRules'
+
+import {source, kapacitorRules} from 'test/resources'
+
+const setup = () => {
+  const props = {
+    source,
+    rules: kapacitorRules,
+    hasKapacitor: true,
+    loading: false,
+    onDelete: () => {},
+    onChangeRuleStatus: () => {},
+  }
+
+  const wrapper = shallow(<KapacitorRules {...props} />)
+
+  return {
+    wrapper,
+    props,
+  }
+}
+
+describe('Kapacitor.Containers.KapacitorRules', () => {
+  describe('rendering', () => {
+    it('renders KapacitorRules', () => {
+      const {wrapper} = setup()
+      expect(wrapper.exists()).toBe(true)
+    })
+
+    it('renders KapacitorRulesTable', () => {
+      const {wrapper} = setup()
+
+      const kapacitorRulesTable = wrapper.find('KapacitorRulesTable')
```

Comment:
the arguments to find dont need to be strings.  the will work with the variable `find(KapacitorRulesTable)`, but that would require an import :)